perfscale_regression_ci/scripts/scalability/replace_loadedworker_nodes.sh

```diff
@@ -0,0 +1,73 @@
+#/!/bin/bash
+##################################################################################################
+## Auth=svetsa@redhat.com
+## Desription: Script for resizing worker nodes with larger worker nodes
+## Polarion test case: OCP-40557 - Replace loaded worker nodes with larger instances
+## https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-40557
+## Customer related: https://access.redhat.com/support/cases/#/case/02864921
+## Pre-requisites: 
+### 1) Prior to running this script, make sure to run loaded-upgrade Jenkins job 
+### to create a new cluster with one of the below profiles and run cluster-density workload
+### https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/loaded-upgrade/
+### Cluster with 3 master nodes and 30 worker nodes using one of these profiles:
+#### AWS: IPI-on-AWS-OVN
+#### GCP: IPI-on-GCP-OVN
+#### Alicloud: IPI-on-AlibabacloudOVN
+### 2) Take a Grafana/Dittybopper snapshot 5 to 10 min after workload has finished.
+### The above snapshot will be the resource baseline measurements for the test.
+## kube-burner config: 
+### openshift_scalability/scripts/replace_nodes/
+### e2e-benchmarking/workloads/kube-burner/workloads/cluster-density/cluster-density.yml
+## Parameters: number of JOB_ITERATIONS
+## Arguments to script: 
+### Required: Cloud provider as CLOUD (aws, gcp, alicloud)
+### Optional: Number of worker nodes to create using new machineset as REPLICAS (default value: 15)
+###################################################################################################
+
+source ../custom_workload_env.sh
+source ../common.sh
+source ../../utils/run_workload.sh
+
+# If parameters is set from upstream ci, overwrite params
+export params=(${PARAMETERS:-gcp 15 120})
+
+export CLOUD="${params[0]}"
+export REPLICAS="${params[1]}"
+export JOB_ITERATIONS="${params[2]}"
+
+export WORKLOAD="cluster-density"
+echo "Testing ${WORKLOAD} on $CLOUD using $REPLICAS replicas for $JOB_ITERATIONS job iterations for new machineset."
```

Comment:
This test is not actually run in code, right?, it is run in [Pre-requisites](https://github.com/svetsa-rh/svt/blob/476652ddd49b988eb9d16836a910b53cf2d35fff/perfscale_regression_ci/scripts/scalability/replace_loadedworker_nodes.sh#L8). So let's remove this output to avoid confusion.